.github/workflows/checkForeignAssetsConfig.yml

```diff
@@ -0,0 +1,24 @@
+name: Check tokens list
+
+on:
+  pull_request:
+  push:
+    branches:
+      - development
+      - staging
+      - production
+      - mainnet
+
+jobs:
+  check-token-list:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-node@v3
+        with:
+          node-version: 20
+          cache: "npm"
+      - run: npm ci
+      - run: npm run build --if-present
+      - run: cd core/tokenRegistry && npx ts-node src/scripts/checkForeignAssetsConfig.ts
```

Comment:
could/should we add this to the existing one so we don't have to wait for it to rebuild another env?